### PR TITLE
[Snyk] Security upgrade @rails/webpacker from 4.3.0 to 5.2.0

### PR DESCRIPTION
**Pull request by its-a-lisa-at-work**
<h3>Snyk has created this PR to fix one or more vulnerable packages in the `yarn` dependencies of this project.</h3>

merge advice

#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - package.json
    - yarn.lock

##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
high severity  |  **661/1000**  <br/> **Why?** Recently disclosed, Has a fix available, CVSS 7.5  | Regular Expression Denial of Service (ReDoS) <br/>[SNYK-JS-SSRI-1085630](https://snyk.io/vuln/SNYK-JS-SSRI-1085630) |  Yes  | No Known Exploit 

(*) Note that the real score may have changed since the PR was raised.

Check the changes in this PR to ensure they won't cause issues with your project.

------------

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  
🧐 [View latest project report](https://app.snyk.io/org/tts/project/5f0f5410-719d-4ceb-a043-651db36f0e71)

🛠 [Adjust project settings](https://app.snyk.io/org/tts/project/5f0f5410-719d-4ceb-a043-651db36f0e71/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"a573911f-af5a-4afc-88b6-457ea6f487db","dependencies":[{"name":"@rails/webpacker","from":"4.3.0","to":"5.2.0"}],"packageManager":"yarn","projectPublicId":"5f0f5410-719d-4ceb-a043-651db36f0e71","projectUrl":"https://app.snyk.io/org/tts/project/5f0f5410-719d-4ceb-a043-651db36f0e71?utm_source=github&utm_medium=fix-pr","type":"auto","patch":[],"vulns":["SNYK-JS-SSRI-1085630"],"upgrade":["SNYK-JS-SSRI-1085630"],"isBreakingChange":true,"env":"prod","prType":"fix","templateVariants":["updated-fix-title","priorityScore","merge-advice-badge-shown"],"priorityScoreList":[661]})

### CODE REVIEW

**Comment by ondrae** [comment]
Looks like we need to bump up node in the test env https://github.com/18F/all_sorns/blob/main/.github/workflows/test.yml#L41

**Comment by folksgl** [comment]
The Gemfile will need to be updated as well to not include the offending webpacker version, correct @ondrae?

**Comment by ondrae** [comment]
I bumped the version of node and webpacker. I don't know what version of node we want to be at, 12.13.1 is just what I happened to have locally. Feel free to change it if you know of a better choice.

**Comment by folksgl** [comment]
@ondrae The latest version of Node looks like 14.16.0. Is there a compelling reason not to use that? Would buy us some time before we have to update again if it passes all the tests. If we don't mind accepting minor version bumps on node we can also just specify '14' according to https://github.com/actions/setup-node

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v1
         with:
-          node-version: 10.13.0
+          node-version: 12.13.1
```
Comment (folksgl) [inline .github/workflows/test.yml:41]: See the "Conversations" tab on the versions available for Node.

**Review by folksgl — CHANGES_REQUESTED** [review]
Can we test with the latest '14' version and see what happens?

**Comment by ondrae** [comment]
Node v14 worked. Ship it!

**Review by folksgl — APPROVED** [review]

**Pull request merged**